src/common/tests/checksum_timing.c

```diff
@@ -212,36 +270,72 @@ m_csum64_finish(struct daos_csummer *obj)
 }
 
 struct csum_ft m_csum64_algo = {
-		.update = m_csum64_update,
-		.get = m_csum64_get,
-		.init = m_csum64_init,
-		.destroy = m_csum64_destroy,
-		.finish = m_csum64_finish,
-		.get_size = m_csum64_get_size,
+		.cf_update = m_csum64_update,
+		.cf_get = m_csum64_get,
+		.cf_init = m_csum64_init,
+		.cf_destroy = m_csum64_destroy,
+		.cf_finish = m_csum64_finish,
+		.cf_get_size = m_csum64_get_size,
 		.name = "mcrc64"
 	};
 #endif
 
+static int
+gurt_64_update(struct daos_csummer *obj,
+	      uint8_t *buf, size_t buf_len)
+{
+
+	uint64_t *value = (uint64_t *)obj->dcs_csum_buf;
+
+	*value = d_hash_murmur64(buf, buf_len, *value);
+	return 0;
+}
+
+struct csum_ft gurt_murmur64_algo = {
+	.cf_update = gurt_64_update,
+	.cf_csum_len = sizeof(uint64_t),
+	.cf_name = "murm64"
+};
+
+static int
+hash_string32_update(struct daos_csummer *obj,
```

Comment:
same comment...

At least maybe make them consistent.  hash_string32* and hash_murmur*

Reply:
changed to be more consistent and hopefully less weird :)